### ISSUE TITLE
Related posts function should use the WC template structure 

### ISSUE THREAD

**Issue by finalwebsites** [bootscore/bs-swiper#81]
Hi,

The swiper integration is really nice!

I was a bit surprised that the function bootscore_related_posts() doesn't use the WC template structure.
While the shortcode [bs-swiper-card-product] is using this structure.

Is there a special reason for this?

**Comment by finalwebsites** [comment]
Right now there is a different approach for the related posts (template) and the other examples (shortcode).
I don't know why this is :)
In my current workaround there is one problem, performance. The used query isn't so fast if there are much more products and/or if I query more products at the same time. A taxonomy query like you did for the related posts shortcode would be much better. In my situation would work a function that I can hook into single product template.
On the other site the block editor is coming to us...

**Comment by crftwrk** [comment]
> On the other site the block editor is coming to us...

Yes, that will change everything. Think it makes no sense to implement a new feature which is probably outdated soon. Let‘s wait until the block editor has arrived.
